config/initializers/devise.rb

```diff
@@ -4,7 +4,7 @@
   # The secret key used by Devise. Devise uses this key to generate
   # random tokens. Changing this key will render invalid all existing
   # confirmation, reset password and unlock tokens in the database.
-  config.secret_key = 'd3d10312e9ac9b210b8628a0640b3171d57cc396f0a8cf08eec32eb3f047f724db3f80ebdaef7de2dc0831eb9f8e7f288e21e54f4ff3f1cf0b59439416bc6650'
+  config.secret_key = Rails.env.production? ? ENV['DEVISE_TOKEN'] : `bundle exec rake secret`
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.